.isort.cfg

```diff
@@ -1,2 +1,7 @@
 [settings]
 known_third_party = cv2,gym,matplotlib,numpy,pytest,scipy,setuptools,torch
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+line_length=88
```

Comment:
Is line length 88 still followed? Hasn't that been increased now?

(Not sure)

Reply:
This is what black recommended. PEP8 recommends 79, should I change that?

Reply:
However, they do say that it's fine as long as the team is fine with it :p